lib/core.js

```diff
@@ -52,6 +57,10 @@ function Zone(parentZone, data) {
 Zone.prototype = {
   constructor: Zone,
 
+  path: function () {
+    return (this.parent ? this.parent.path() : '') + '/' + this.$id;
```

Comment:
I kind of like "/" as a path delimiter but let's got " > " as long at it's not "\".